src/components/style.css

```
@@ -358,6 +358,93 @@ ul {
   font-size: 14px;
   color: #878787;
 }
+@media screen and (max-width: 500px) {
+  div.formRoot,
+  select.custom-select,
+  input.form-control {
+    font-size: 0.8rem;
+  }
+  select.custom-select,
+  input.form-control {
+    width: 100%;
+  }
+  div.form-child-left,
+  div.form-child-right {
+    flex: 100%;
+  }
+  div.modal-content {
+    margin: 100px auto;
+    width: 310px;
+    position: relative;
+  }
+}
+@media screen and (min-width: 501px) and (max-width: 577px) {
+  div.formRoot,
+  select.custom-select,
+  input.form-control {
+    font-size: 0.7rem;
+  }
+  select.custom-select,
+  input.form-control {
+    width: 80%;
+  }
+  div.form-child-left,
+  div.form-child-right {
+    flex: 50%;
+  }
+  div.modal-content {
+    margin-top: 100px;
```

```suggestion
    margin-top: 300px;
```

The heading is missing in this range. Please make the suggested change.